modules/apps/transfer/types/transfer_authorization_test.go

```diff
@@ -389,6 +389,84 @@ func (suite *TypesTestSuite) TestTransferAuthorizationAccept() {
 				suite.Require().Nil(res.Updated)
 			},
 		},
+		{
+			"success - allowed forwarding hops",
+			func() {
+				msgTransfer.Forwarding = types.NewForwarding("", types.Hop{PortId: "1", ChannelId: "channel-1"}, types.Hop{PortId: "2", ChannelId: "channel-2"})
+				transferAuthz.Allocations[0].AllowedForwardingHops = []*types.Hops{
+					{
+						Hops: []types.Hop{
+							{PortId: "1", ChannelId: "channel-1"},
+							{PortId: "2", ChannelId: "channel-2"},
+						},
+					},
+				}
+			},
+			func(res authz.AcceptResponse, err error) {
+				suite.Require().NoError(err)
+				suite.Require().True(res.Accept)
+			},
+		},
+		{
+			"failure - allowed forwarding hops is longer",
```

Comment:
Maybe something like this?

```suggestion
			"failure - allowed forwarding hops contains more hops",
```

Reply:
Done!